src/model/attributes/relations/MorphTo.ts

```diff
@@ -196,14 +196,14 @@ export class MorphTo extends Relation {
   /**
    * Check if related model is new
    */
-   private $isNewRelated(type: string): boolean {
+  private $isNewRelated(type: string): boolean {
```

Comment:
We can remove `$` prefix. The reason why I'm prefixing model methods with `$` is that it might conflict with user defined model field or methods.

Other than models, it's just fine to use method names without `$` 👍

Reply:
Thanks!